ext/com_dotnet/com_com.c

```diff
@@ -697,7 +694,7 @@ PHP_FUNCTION(com_event_sink)
 
 	RETVAL_FALSE;
 
-	if (FAILURE == zend_parse_parameters(ZEND_NUM_ARGS(), "Oo|z/",
+	if (FAILURE == zend_parse_parameters(ZEND_NUM_ARGS(), "Oo|z!/",
```

Comment:
Shouldn't this be changed to accept array|string|null as well?

Reply:
Isn't `/` separates the argument? Can I still use the related fast ZPP macro?

Reply:
It doesn't looks like the code actually needs separation. However, I think the com::__construct implementation above needs separation but doesn't do it, as it converts array values in place. @cmb69 might want to check that.

Reply:
Yes, `server_params` needs to be separated, or we use `zval_get_string()` for the strings.

Reply:
See PR #6040 for the "master" branch. For PHP 7 I'd do the separation.

Reply:
I'll commit all the review fixes as soon as the linked PR is merged, so that I can rebase to master. :)

Reply:
I have just merged that PR. Thanks for the reminder! :)